src/api/archive_unstable_storageDiff.md

```diff
@@ -0,0 +1,98 @@
+
+- `previousHash` (optional): String containing a hexadecimal-encoded hash of the header of the block from which the storage difference will be calculated. The `previousHash` must be an ancestor of the provided `hash`.  When this parameter is provided, the storage difference is calculated between the `hash` block and the `previousHash` block. If this parameter is not provided, the storage difference is calculated between the `hash` block and the parent of the `hash` block.
+
+- `items` (optional): Array of objects. The structure of these objects is found below.
+
+    ```json
+    "items": [
+        {
+            "prefixes": [
+                {
+                    "key": "0x...",
+                    "type": "value" | "hash" | "none",
+                },
+            ],
+
+            "trieType": "mainTrie" | "childTrie",
+            "childTrieKey": "0x...",
+        },
+    ]
+    ```
+
+  If the `items` parameter is not provided, the storage difference is calculated for all keys of the main storage trie.
+
+  Each element in `items` must be an object containing the following fields:
+
+  - `prefixes` (optional): Array of JSON objects describing how the storage difference will be calculated. Each object contains the following fields:
+    - `key`: String containing a hexadecimal-encoded key prefix. Only the storage entries whose key starts with the provided prefix are returned.
+    - `type`: String equal to one of: `value`, `hash`, `none`.
+
+  - `trieType`: A string indicating the trie type for which the storage difference will be calculated. The possible values are:
+    - `mainTrie`: The storage difference is calculated for the main storage.
+    - `childTrie`: The storage difference is calculated for the child trie of the "default" namespace. If this type is provided, the `childTrieKey` field must be provided.
+
+  - `childTrieKey`: String containing the hexadecimal-encoded key of the child trie of the "default" namespace. This field is only present when the `trieType` field is set to `childTrie`.
+
+**Return value**: A JSON object.
+
+The JSON object returned by this function has the following format:
+
+```json
+{
+    "result": [
+        {
+            "differences": [
+                {
+                    "key": "0x...",
+                    "value": "0x...",
+                    "hash": "0x...",
+                    "type": "added" | "modified" | "deleted",
+                },
+            ],
+
+            "trieType": "mainTrie" | "childTrie",
+            "childTrieKey": "0x...",
+        }
+    ],
+}
+```
+
+The `result` field contains an array of objects, each containing a JSON object:
+
+The `differences` field is an array of objects describing storage differences. Each element contains the following fields:
+
+- `key`: String containing the hexadecimal-encoded key of the storage entry. If the key prefix was provided in the `items` parameter, the `key` field is guaranteed to start with one of the key prefixes provided.
```

Comment:
> If the key prefix was provided in the `items` parameter, the `key` field is guaranteed to start with one of the key prefixes provided.

What if the key prefix was not provided in the `items` parameter?

Or perhaps this is aiming for something more like: "String containing the hexadecimal-encoded key of the storage entry. A prefix of this key will have been provided in the `items` input"? I might not 100% understand!

Reply:
Yep the suggestion clarifies things, thanks!